config/api.yml

```diff
@@ -1443,6 +1443,26 @@
       :get:
       - :name: read
         :identifier: event_streams_show
+  :event_definition_sets:
+    :description: Event Definition Sets
+    :identifier: policy_simulation
+    :options:
+    - :collection
+    :verbs: *gp
+    :klass: MiqEventDefinitionSet
+    :subcollections:
+    - :events
+    :collection_actions:
+      :get:
+      - :name: read
+        :identifier: policy_simulation 
```

Comment:
I don't understand the `policy_simulation` choice.   @gtanzillo Thoughts?

Reply:
This endpoint will be used to seed some dropdowns in the policy simulation, of course if https://github.com/ManageIQ/manageiq/issues/19110 is sorted out :trollface: , it can get its own feature (I guess)

Reply:
ManageIQ/manageiq#19110 is going to makes things a lot more reasonable I think 😃